### ISSUE TITLE
reverse generate Gauge graph

### ISSUE THREAD

**Issue by woodmanhu** [bernii/gauge.js#144]
Is there a way that I can set max on the left and 0 to the right? like from left to right the graphic will show:
 {100-50: red}, {50-30: yellow},{30-0: green}?

**Comment by kplindegaard** [comment]
As far as I remember, increasing values generate a clockwise rotation of the gauge pointer and there is no flag or option to reverse that in the internal calculations. 

However, you can probably fake it very easily: Use negative numbers. What I mean is that pass in -1.0*value instead of just value. Worth trying :)
Remember to set min and max limits to -100 and 0 respectively.

**Comment by woodmanhu** [comment]
Good idea! I didn't know it can take negative numbers. I'll try it on my project. Thanks!

**Comment by woodmanhu** [comment]
It does work if everything's set correctly, but the static labels will display negative numbers too.

**Comment by kplindegaard** [comment]
Yes, that's right. Can't fool the static labels.... ;) Next step is to create your own number formater where you employ the same trick or a Math.abs(). 
Request for a custom number formatting function is described in #122, but that's not implemented yet. You thus have to edit the source code yourself as of now. It's there in the formatNumber function in gauge.js if you care to have a go.

**Comment by woodmanhu** [comment]
I dig though the code and from my use perspective of this plug-in, I can keep the graph always take negetive numbers and always display the range between -100 to 0. So I modified the min.js source code a little bit:

Original:
 var t, i, e;
  return i = 1 <= arguments.length ? d.call(arguments, 0) : [], e = i[0], t = i[1],r(e.toFixed(t)) 
  },
Replaced to:
 var t, i, e;
  return i = 1 <= arguments.length ? d.call(arguments, 0) : [], e = i[0]***(-1)**, t = i[1],r(e.toFixed(t)) 
  },

And it displays the positive numbers on the graph.

**Comment by kplindegaard** [comment]
There you go :)

**Comment by woodmanhu** [comment]
Thank you again for your good idea. Plus by the way this is a great tool.

**Comment by NiceGuyNimni** [comment]
I want to contribute to this project and I thought about adding support for reversing the gauge.
I'm new to contributing but not to coding, should I just do it and add a pull request or are there other steps I'm missing?
